### ISSUE TITLE
[A380X] Flypad services page causes doors to open/close in flight

### ISSUE THREAD

**Issue by StylishKitsune** [flybywiresim/aircraft#9216]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2024-10-31T15:54:41+00:00",
    "ref": "refs/tags/v0.12.0",
    "sha": "2fe89113291f2478347dde0dc3228b55dc924f74",
    "actor": "FoxtrotSierra6829",
    "event_name": "manual",
    "pretty_release_name": "stable/v0.12.0",
    "version": "a380x-v0.12.0-rel.2fe8911"
}
```

### Describe the bug

Switching to/from the services page on the Flypad causes the doors to cycle between open/closed during flight

### Expected behavior

Doors stay closed when switching to services page

### Steps to reproduce

1. Switch to Services page in flypad during cruise
2. Doors switch from open to closed or closed to open
3. Confirm in external view
4. Switch to fuel page in flypad
5. Switch back to Services page and doors switch again

### References (optional)

Video of issue:

https://www.youtube.com/watch?v=qRAiKmJ-LCs

### Discord Username (optional)

StylishKitsune

**Comment by TxmoTK** [comment]
Had the same error

**Comment by aubinschneider13** [comment]
same

**Comment by cptalpdeniz** [comment]
this is %100 reproducible for me. Anytime I go to Services page in the air, the doors randomly open.

**Comment by Benjozork** [comment]
Fixed by #9104, slated for stable v0.12.1
